samples/remote-secret-partial-update.yaml

```
@@ -0,0 +1,17 @@
+# this can be used to update the data for the test-remote-secret defined in
+# the remote-secret.yaml.
+# Applying this secret to the cluster will NOT create a new RemoteSecret 
+# object, it can only associate data with a pre-existing one.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-remote-secret-secret
+  labels:
+    appstudio.redhat.com/upload-secret: remotesecret
+  annotations:
+    appstudio.redhat.com/remotesecret-name: test-remote-secret
+    appstudio.redhat.com/remotesecret-partial-update: true
```

```suggestion
    appstudio.redhat.com/remotesecret-partial-update: "true"
```
This needs to be string otherwise kubectl complains